engine/table/src/main/java/io/deephaven/engine/table/impl/by/PartitionByChunkedOperator.java

```diff
@@ -90,13 +91,17 @@ public interface AttributeCopier {
     private final String callSite;
 
     private final ObjectArraySource<QueryTable> tables;
-    private final ObjectArraySource<WritableRowSet> addedRowSets;
-    private final ObjectArraySource<WritableRowSet> removedRowSets;
-    private final ObjectArraySource<WritableRowSet> modifiedRowSets;
+    private final ObjectArraySource<RowSetBuilderSequential> addedBuildersSequential;
+    private final ObjectArraySource<RowSetBuilderRandom> addedBuilders;
```

Comment:
We could get away with just one source, here, if we aren't offended by casting on retrieval. It wouldn't be the only time we ever did such a thing. That would avoid some unnecessary re-allocation of arrays, etc.